chef/roles/deployer-client.rb

```diff
@@ -2,11 +2,7 @@
 name "deployer-client"
 description "Deployer Client role - Discovery components"
 run_list(
-         "recipe[barclamp]",
-         "recipe[repos]",
-         "recipe[crowbar-hacks]",
-         "recipe[ohai]",
-         "recipe[kernel-panic]"
+         "recipe[deployer::role_deployer_client]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.